specs/trust-store-trust-policy.md

```diff
@@ -242,16 +313,32 @@ Signature verification levels provide defined behavior for each validation e.g.
     The trust policy with global scope applies to all the artifacts.
     There can only be one trust policy that uses a global scope.
 
-#### Selecting a trust policy based on artifact URI
+##### Blob Trust Policy Constraints
+- There MUST be only one trust policy with `wildcardPolicy` set to `true`
+- `signatureVerification` MUST not be set to `skip` if the policy is marked as a wildcard policy
 
-- For a given artifact there MUST be only one applicable trust policy, except for trust policy with global scope.
-- For a given artifact, if there is no applicable trust policy then implementations of the [Notary Project verification specification](./signing-and-verification-workflow.md) MUST consider the artifact as untrusted and fail signature verification.
+#### Selecting a trust policy to verify a signed OCI artifact
+
+- The URI of an OCI artifact dictates the trust policy that gets applied to verify the artifact's signature.
+- For a given OCI artifact there MUST be only one applicable trust policy, except for trust policy with a global scope.
+- For a given OCI artifact, if there is no applicable trust policy then implementations of the [Notary Project verification specification](./signing-and-verification-workflow.md) MUST consider the artifact as untrusted and fail signature verification.
 - The scope MUST NOT support reference expansion i.e. URIs must be fully qualified.
   E.g. the scope should be `docker.io/library/registry` rather than `registry`.
 - Evaluation order of trust policies:
   1. *Exact match*: If there exists a trust policy whose scope contains the artifact's repository URI then the aforementioned policy MUST be used for signature evaluation.
      Otherwise, continue to the next step.
-  1. *Global*: If there exists a trust policy with global scope then use that policy for signature evaluation.
+  1. *Global*: If there exists a trust policy with global/wildcard scope (`*`) then use that policy for signature evaluation.
+     Otherwise, fail the signature verification.
+
+#### Selecting a trust policy to verify a signed blob
+
+- The signature verifier must select the appropriate trust policy for blob signature verification using policy names.
+- For a given policy name selected by the verifier, there MUST be only one trust policy with that exact name.
+- For a given policy name selected by the verifier, if there is no matching trust policy with that name, then implementations of the [Notary Project verification specification](./signing-and-verification-workflow.md) MUST consider the blob as untrusted and fail signature verification.
+- Evaluation order of trust policies:
+  1. *Exact match*: If there exists a trust policy whose name exactly matches the one provided by the signature verifier then the aforementioned policy MUST be used for signature evaluation.
+     Otherwise, continue to the next step.
```

Comment:
Given the discussion here: https://github.com/notaryproject/specifications/pull/283#discussion_r1475011602
When there isn't an exact match, we should not `continue to the next step`. Instead, `fail the signature verification` is the expected behavior?
```suggestion
     Otherwise, fail the signature verification.
```